pkgs/top-level/python-aliases.nix

```diff
@@ -453,6 +453,7 @@ mapAliases ({
   querystring_parser = querystring-parser; # added 2024-01-07
   qcodes-loop = throw "qcodes-loop has been removed due to deprecation"; # added 2023-11-30
   qiskit-aqua = throw "qiskit-aqua has been removed due to deprecation, with its functionality moved to different qiskit packages";
+  qrcode_terminal = qrcode-terminal; # added 2024-05-26
```

Comment:
```suggestion
```
alias is needed when a package name is changed on nixpkgs. Since this is a new package, you do not need to add an alias.